### DISCUSSION TITLE
Display only works after typing 'startx' in the terminal

### DISCUSSION THREAD

**Discussion by Ejobat** [minios-linux/minios-live#36] [Q&A]
After changing the password for the 'live' and 'root' users, as well as changing the username for the 'live' user, I've noticed that the display (X server) only works for the 'live' user if I type the 'startx' command, or if I log in as the 'root' user and then switch back to the 'live' user. How to fix this?

**Comment by crim50n** [comment]
User names should be set in the minios.conf file before the system boots for the first time. Is the problem still relevant?
